SignalMessaging/utils/OWSWindowManager.m

```diff
@@ -14,16 +14,23 @@
 
 const CGFloat OWSWindowManagerCallBannerHeight(void)
 {
-    if ([UIDevice currentDevice].isIPhoneX) {
-        // On an iPhoneX, the system return-to-call banner has been replaced by a much subtler green
-        // circle behind the system clock. Instead, we mimic the old system call banner as on older devices,
-        // but it has to be taller to fit beneath the notch.
-        // IOS_DEVICE_CONSTANT, we'll want to revisit this when new device dimensions are introduced.
-        return 64;
-    } else {
+    if (@available(iOS 11.4, *)) {
+        return CurrentAppContext().statusBarHeight + 20;
+    }
+
 
+    if (![UIDevice currentDevice].hasIPhoneXNotch) {
         return CurrentAppContext().statusBarHeight + 20;
     }
+
+    // Hardcode CallBanner height for iPhone X's on older iOS.
+    //
+    // As of iOS11.4 and iOS12, this no longer seems to be an issue, but previously statusBarHeight returned
```

Comment:
Because the statusBarHeight behavior seems to have changed for notched devices, this hardcoding is no longer necessary. 

As such, this change isn't urgent, since our default non-iphoneX behavior should work - at least it does with my physical iPhoneX on iOS12, the iPhoneX 11.4 simulator, and all the new iOS12 X* simulators.